addons/binding/org.openhab.binding.network/ESH-INF/thing/thing-types.xml

```diff
@@ -69,5 +78,11 @@
         <category>Number</category>
 	    <state readOnly="true"></state>
     </channel-type>
+    <channel-type id="last_seen">
+    	<item-type>DateTime</item-type>
+    	<label>Last seen</label>
+    	<description>States the last time the device has been seen online</description>
+    	<category>DateTime</category>
+    	<state readOnly="true"></state></channel-type>
```

Comment:
I'm sorry about the formatting and will fix it. I'm new to OpenHAB development and will try to comply to the rules, but it's a bit much all at once.